src/python/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+cmake_minimum_required(VERSION 3.18)
+
+# [DLIS-7232] tritonfrontend package expects all supported packages to be
+# built, without any check/verification for respective frontend enable flags.
+# Support for partial builds(ex: HTTP but not gRPC) will be addressed later.
+if(NOT (${TRITON_ENABLE_HTTP} AND ${TRITON_ENABLE_GRPC}))
+  message("tritonfrontend python package build skipped when relevant frontends are disabled.")
+  message("In order to build tritonfrontend, set -DTRITON_ENABLE_HTTP=ON -DTRITON_ENABLE_GRPC=ON")
+  return()
+endif()
```

Comment:
Is this block actually required?

Reply:
The purpose of this block is to provide visibility during the make process on why the `tritonfrontend` package cannot be built without certain flags enabled. Support for conditional building is to be addressed in a follow-up PR.

Reply:
Ok then it means that target is not included and CMake will ignore it's build. 
Why do we need to notify anyone about it ? In other words if it something that users must look by default, then make it enabled by default.

Reply:
Thank you for suggestion! Moved the `message()` statements to before the return statement.